src/main/java/uk/nhs/digital/nhsconnect/lab/results/mesh/MeshService.java

```diff
@@ -118,7 +118,7 @@ private void processSingleMessage(String messageId) {
                     "Downloaded MeshMessageId={} has an unsupported MeshWorkflowId={}."
                         + "It is not a {} or {} message, therefore it has been left in the inbox.",
                     meshMessage.getMeshMessageId(), meshMessage.getWorkflowId(),
-                    MessageType.PATHOLOGY.getCode(), MessageType.SCREENING.getCode()
+                    MessageType.PATHOLOGY_VARIANT_3.getCode(), MessageType.SCREENING.getCode()
```

Comment:
Can we add NHS002 to this log as well?

Reply:
I've reworked it a little so it's derived from the collection of supported types.